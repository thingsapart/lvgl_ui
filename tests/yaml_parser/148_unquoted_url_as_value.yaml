# A URL is a common unquoted string value that contains a colon.
# This must be parsed as a single string.
endpoint: https://api.example.com/v1/users
