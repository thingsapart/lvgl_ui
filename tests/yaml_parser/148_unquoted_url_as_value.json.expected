[{
		"endpoint":	"https://api.example.com/v1/users"
	}]
